integration-tests/pkg/mock_sensor/server.go

```diff
@@ -32,7 +32,6 @@ const (
 // us to use any comparable type as the key)
 type ProcessMap map[types.ProcessInfo]interface{}
 type LineageMap map[types.ProcessLineage]interface{}
-type ConnMap map[types.NetworkInfo]interface{}
```

Comment:
The changes in this file were made here https://github.com/stackrox/collector/pull/1902/files